landing-page/layouts/_default/single.html

```diff
@@ -17,8 +17,16 @@
 </head>
 {{ partial "header.html" . }}
 <body dir="{{ .Site.Language.LanguageDirection | default " ltr" }}">
-    <div class="grid-container">
-        {{ partial "toc.html" . }}
+        {{ if (.Params.toc) }}
+        <div class="grid-container" style="--grid-dimensions: 5fr 1fr">
```

Comment:
This grid-container makes the page content and TOC align much better and makes the page more responsive overall.